x-pack/plugins/security_solution/server/lib/machine_learning/index.test.ts

```diff
@@ -104,4 +105,20 @@ describe('getAnomalies', () => {
       ])
     );
   });
+
+  it('ignores anomalies that do not have finalized scores', () => {
+    const mockMlAnomalySearch = jest.fn();
+    getAnomalies(searchParams, mockMlAnomalySearch);
+    const filters = getFiltersFromMock(mockMlAnomalySearch);
+
+    expect(filters).toEqual(
+      expect.arrayContaining([
+        {
+          term: {
+            is_interim: false,
+          },
+        },
+      ])
```

Comment:
FYI, this means that alerts won't fire for anomalies until after the bucket has been completed. 

For short buckets, this is cool (like, `15m`). But if there are jobs with 1hr+ length buckets, it means an anomaly won't trigger this alert until after a whole hour.

Reply:
We're going to meet with @randomuserid  and the @elastic/protections folks to review the existing ML Job bucket times, and corresponding Rule's interval + lookback to ensure we don't have any gaps here. I believe @randomuserid took this all into account when initially developing the jobs/rules, but we'll ensure to do an audit here to make sure everything is 👍.

Thanks again for all your help and feedback around debugging this issue, really grateful for your feedback 🙂

Reply:
Most of the jobs have 15m buckets. I set the loopbacks are with the assumption that anomaly may take 15-30 minutes to be picked up by the rules depending on where the bucket is when the rule fires. Some, like CloudTrail, have longer loopbacks, because those pipelines have much more variable latency.